scripts/qiita

```diff
@@ -452,7 +452,12 @@ def start(port):
     # having to do that. The solution will (perhaps) be to move this subcommand
     # into an entirely different script.
     from qiita_pet.webserver import Application
+    from tornado.options import options, parse_command_line
 
+    if _CONFIG.log_path:
```

Comment:
I just check the file and the variable _CONFIG is only used here. This is incorrect, as we need to do `from qiita_core.settings import qiita_config`.